.travis.yml

```diff
@@ -1,48 +1,32 @@
 language: python
-cache: pip
-dist: trusty
+dist: xenial
+
+
+cache:
+  pip: true
+  directories:
+    - $HOME/.cache/pip
+    - $HOME/virtualenv
 
 env:
   global:
     - DJANGO_SETTINGS_MODULE="tests.app.settings"
 
-
-matrix:
-  include:
-    - env: TOXENV=py35-dj20-wt24
-      python: 3.5
-    - env: TOXENV=py35-dj21-wt24
-      python: 3.5
-    - env: TOXENV=py36-dj20-wt24
-      python: 3.6
-    - env: TOXENV=py36-dj21-wt24
-      python: 3.6
-    - env: TOXENV=py35-dj20-wt25
-      python: 3.5
-    - env: TOXENV=py35-dj21-wt25
-      python: 3.5
-    - env: TOXENV=py36-dj20-wt25
-      python: 3.6
-    - env: TOXENV=py36-dj21-wt25
-      python: 3.6
-    # - env: TOXENV=py36-dj22-wt25
-      # python: 3.6
-    - env: TOXENV='flake8,isort'
```

Comment:
Can you add this line back in for testing flake8 + isort 

Some of your files that have `import wagtail` at the top will fail the isort test

Reply:
wait nevermind I see it's added in tox.ini